watertap/property_models/NaCl_prop_pack.py

```diff
@@ -702,9 +712,8 @@ def _osm_coeff(self):
 
         def rule_osm_coeff(b):
             return b.osm_coeff == (
-                b.params.osm_coeff_param["2"]
-                * b.mass_frac_phase_comp["Liq", "NaCl"] ** 2
-                + b.params.osm_coeff_param["1"] * b.mass_frac_phase_comp["Liq", "NaCl"]
+                b.params.osm_coeff_param["2"] * b.mass_frac_phase_comp[p, "NaCl"] ** 2
+                + b.params.osm_coeff_param["1"] * b.mass_frac_phase_comp[p, "NaCl"]
```

Comment:
If the rule name does not have a "_phase" (e.g. it does not have self.params.phase_list as a variable) in its name, should I still replace the ["Liq"] with [p]?